.pre-commit-config.yaml

```diff
@@ -59,6 +59,17 @@ repos:
         - flake8==5.0.4
         - flake8-bugbear==22.7.1
         - pandas-dev-flaker==0.5.0
+    # Added pylint, at the moment it is excluding a lot of messages
+-   repo: https://github.com/pycqa/pylint
+    rev: v2.15.3
+    hooks:
+    -   id: pylint
+        types: [python]
+        args: 
+            [
+                "-rn",
+                "-d=C,R,W,import-error,no-member,no-name-in-module,unsubscriptable-object,invalid-unary-operand-type,unexpected-keyword-arg,not-an-iterable,unsupported-assignment-operation,unsupported-membership-test,no-value-for-parameter,unpacking-non-sequence,bad-super-call,access-member-before-definition,used-before-assignment,too-many-fucntion-args,abstract-class-instantiated,not-callable,too-many-fucntion-args,invalid-repr-returned"
+            ]
```

Comment:
can these be added to `pyproject.toml`? something like
```
[tool.pylint.messages_control]
disable = [
  "import-error",
  "no-member",
]
```
(and all the others in there)

Reply:
Done